src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -213,15 +216,25 @@ private string ValidateGitSource(string gitSource)
             return gitSource;
         }
 
-        private (string ProjectVersion, string FallbackVersion, string GitSource) ValidateCompareToDashboard(string projectVersion, string fallbackVersion, string gitSource)
+        private (string ProjectVersion, string FallbackVersion, string GitSource, IEnumerable<FilePattern> DashboardCompareFileExcludePatterns) ValidateCompareToDashboard(string projectVersion, string fallbackVersion, string gitSource, IEnumerable<string> dashboardCompareFileExcludePatterns)
```

Comment:
I don't think the exclude from diff result is exclusive to dashboard compare so it should have it's own validator